generic/piranha-tree-sitter/run_piranha_demo.sh

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2022 Uber Technologies, Inc.
+
+# <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a copy of the License at
+# <p>http://www.apache.org/licenses/LICENSE-2.0
+
+# <p>Unless required by applicable law or agreed to in writing, software distributed under the
+# License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+cargo build --release
```

Comment:
Get rid of unused functions:

-----------

warning: associated function is never used: `code`
   --> src/piranha.rs:527:10
    |
527 |   pub fn code(&self) -> String {
    |          ^^^^
    |
    = note: `#[warn(dead_code)]` on by default

warning: associated function is never used: `path`
   --> src/piranha.rs:531:10
    |
531 |   pub fn path(&self) -> &PathBuf {
    |          ^^^^